### ISSUE TITLE
Request: Add Musllinux wheels to workflow (PEP 656 approved)

### ISSUE THREAD

**Issue by DeadEnded** [mark9064/pyamdgpuinfo#7]
Hello again!

Back when I was making my custom component, we had some discussions where I learned about the wheels, etc.
You had mentioned there was a PEP for supporting musl wheels https://github.com/mark9064/pyamdgpuinfo/issues/3#issuecomment-805107151.

... and it was approved a while ago https://github.com/pypa/auditwheel/issues/305.

Recently the program I build my component for updated to python 3.10, and with the support of musllinux wheels they have archived their custom wheels as they are no longer need.  This means my component is temporarily broken! Luckily there is an easy fix... I need musllinux wheels to be built and publish along with the manylinux wheels.

SO - I know that doesn't sound difficult as in theory it is a workflow update (and possibly debugging any issues).  I took a look at your workflow to see if I could help and do a PR to get them added... I followed your process flow and see where you use a container from `RalfG`... and in that repo there is a request to add musllinux support https://github.com/RalfG/python-wheels-manylinux-build/pull/52.

So long story short... can you look into adding musllinux wheels to your workflow?
I understand that potentially you will want to wait for `RalfG` to add support so you can use a similar process as your manylinux flow.
If there is anything I can do to help, let me know and I will see what I can do!

Thanks!
Deadend

**Comment by mark9064** [comment]
Hi again! Good to see that musl wheels have progressed along to being possible now. I'd definitely like to build them, but as you rightly spotted it looks like the CI action I'm using doesn't support it yet. The PR has been there since mid June which isn't too encouraging. Maybe I'll end up switching to a different script, not sure yet.

Thanks for the heads up and research, will keep you updated on where I get with this. There might be further issues as I've never tried this before, but hopefully it's doable :D

**Comment by mark9064** [comment]
https://github.com/RalfG/python-wheels-manylinux-build/pull/52 just got merged so hopefully I can get this working soon

**Comment by DeadEnded** [comment]
Fantastic!
Looks like this might not be too far away then!

**Comment by mark9064** [comment]
Should be available with the 2.1.3 release! Let me know how it goes

**Comment by DeadEnded** [comment]
> 

Looks great!
Just restarted my container, and it installed without any issues.
Awesome job!
